src/SDK/MonitoringApplianceEmulator/ViewModels/MetricChartPropertyControlViewModel.cs

```diff
@@ -0,0 +1,213 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="MetricChartPropertyControlViewModel.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.ViewModels
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Globalization;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using System.Windows;
+    using System.Windows.Media;
+    using LiveCharts;
+    using LiveCharts.Configurations;
+    using LiveCharts.Defaults;
+    using LiveCharts.Wpf;
+    using Microsoft.Azure.Monitoring.SmartDetectors.Metric;
+    using Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.Controls;
+    using Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.Models;
+    using Microsoft.Azure.Monitoring.SmartDetectors.RuntimeEnvironment.Contracts.AlertProperties;
+
+    /// <summary>
+    /// The view model class for the <see cref="MetricChartPropertyControl"/> control.
+    /// </summary>
+    public class MetricChartPropertyControlViewModel : ObservableObject
+    {
+        private static readonly Brush SeriesColor = Brushes.DeepSkyBlue;
+
+        private readonly ObservableTask<ChartValues<DateTimePoint>> loadChartTask;
```

Comment:
>private readonly ObservableTask<ChartValues<DateTimePoint>> loadChartTask; [](start = 8, length = 74)

expose this as a public property, and bind a busy animation to its state

Reply:
See how I used the loading animation when loading subscriptions, RGs etx

---
In reply to: [304378426](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/70#discussion_r304378426) [](ancestors = 304378426)

Reply:
after doing it I would like to see how it works 

---
In reply to: [304379533](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/70#discussion_r304379533) [](ancestors = 304379533,304378426)